### ISSUE TITLE
dcc.Relayout not working with dcc.Graph 's clickData  

### ISSUE THREAD

**Issue by ArshiyanAlam** [jimmybow/mydcc#2]
I have taken the example in README.md and added a callback with clickData at the end. Also debug is set to True.
On running this page becomes irresponsive. 
```
import dash
import dash_core_components as dcc
import dash_html_components as html
from dash.dependencies import Input, Output, Event, State
import mydcc

app = dash.Dash()
app.layout = html.Div([
    mydcc.Relayout( id = "rrr", aim = 'graph' ),
    dcc.Graph( id = 'graph',
               figure = { 'data': [  {'x': [1, 2, 3], 'y': [4, 1, 2]}  ]  }
              ),
    dcc.Input(id = 'in',
              placeholder = 'Enter a title ...',
              type = 'text',
              value = ''  
              ),
    html.Div('xxx',id='view')
])

@app.callback(
    Output('rrr', 'layout'),
    [Input('in', 'value')])
def myfun(x):
    return {'title':x}


@app.callback(
    Output('view', 'children'),
    [Input('graph', 'clickData')])
def bStoreEditor(clickData):
    print(clickData)
    return 'yyy'



if __name__ == '__main__':
    app.run_server(debug=True)
```

**Comment by jimmybow** [comment]
.... for some reason it render again and again, I will try how to improve it @@ 
https://github.com/jimmybow/mydcc/blob/master/src/components/Relayout.react.js

**Comment by jimmybow** [comment]
the problem fixed in version 0.1.22.
thanks for your feedback
